.gitignore

```diff
@@ -1,8 +1,3 @@
-.terraform
-crash.log
-examples/terraform.tfstate
-**/*/*.tfstate.backup
 dist/
-terraform-provider-firehydrant
-.tool-versions
-.idea
+local/bin/terraform-provider-firehydrant
+local/examples/**/*
```

Comment:
can we keep those other terraform files in this ignore file and just add the new stuff to the bottom?

Reply:
I'd prefer to remove them so its clear when you're creating files that might get checked back in -- everything is fair game in `local` and everywhere else we want to let you know you created something that might bite you later, right?